products/pan-gp.md

```diff
@@ -69,6 +70,7 @@ releases:
     releaseDate: 2017-01-30
     latest: "4.0"
     latestReleaseDate: 2017-01-30
+    link: https://docs.paloaltonetworks.com/globalprotect/4-0/globalprotect-app-release-notes/gp-app-release-information/features-introduced-in-gp-app
```

Comment:
I get a 404 on this link

Reply:
Looks like it has been removed 
![image](https://github.com/endoflife-date/endoflife.date/assets/5235127/135a5f86-f566-4f31-9542-4414a93a2576)